### ISSUE TITLE
Replace reference to adodb by late binding

### ISSUE THREAD

**Issue by team-moeller** [team-moeller/better-access-charts#186]
The class module "helpers" uses adodb.connection and adodb.recordset.
Therefore a reference to adodb is necessary.
Replace reference by late binding.

**Comment by team-moeller** [comment]
```
Dim conn1 As Object
Set conn1 = CreateObject("ADODB.Connection") 
```

**Comment by team-moeller** [comment]
Released with version [4.23.07](https://github.com/team-moeller/better-access-charts/releases/tag/4.23.07)
